### DISCUSSION TITLE
Being able to have different filters for the same burrows.

### DISCUSSION THREAD

**Discussion by phooenixdassmatte** [DFHack/dfhack#4302] [Ideas]
I want to have a burrows that restricts childrens movement, and i also have an emergency burrows that is paused and turned on when neccessary. Because the squares on burrows can be very confusing and it's easier to just modify one burrow for the actual area and then manage who gets to be there with the burrows manager. So, one area that can be used by 2 different settings, and in my case the emergency almost always being paused with all adults in it, and the same area but for children unpaused at all times, confining them to the burrow.

**Comment by phooenixdassmatte** [comment]
![df hack burrows](https://github.com/DFHack/dfhack/assets/157321203/a8da45f9-2f5b-4554-889b-af2b820d4a53)

A little example

**Comment by myk002** [comment]
I'm not sure what the best solution is here, but here are a few related notes:
- Are you sure you want an "emergency burrow" instead of a civ alert? You don't have to manage membership for a burrow for a civ alert. All civilians (and pets) will rush there when the civ alert is engaged, regardless of where they are "assigned".
- the `burrow` command has facilities for copying burrow areas and membership between burrows. There's no UI for it, but the commands are quick to run. Docs: https://docs.dfhack.org/en/latest/docs/tools/burrow.html

**Reply by phooenixdassmatte** [reply]
I didn't see anything on the doc about copying and pasting the burrow tiles. Although there might be a semi solution, take some time and make the second children burrow, add the children and then set both with a + in the end, then they would expand automatically both at the same time?

And for the civ alert, i'm currently training marksmen and don't want them engaging the enemy yet, and my captain of the guard is unarmed and very weak (intended) so those could potentially end up dead so i'm sadly not using it. Every time there is an attack i pause and go into burrows and add all civilians to the burrow who might have joined since last time, and noob marksmen and captain of the guard is also there and unpause it.

**Reply by phooenixdassmatte** [reply]
Or is it: 
burrow tiles emergency burrow [burrow tiles childrens area]
?
i have seen the "[" in previous locations in code but i haven't figure out how to do them on my swedish keyboard with a game with english ascii table x)

**Reply by myk002** [reply]
a syntax element surrounded by `[` and `]` means that the element is optional -- you don't type the brackets literally. The command would be:
`burrow tiles set children emergency`
for setting the tiles in the `children` burrow to be the same as the tiles in the `emergency` burrow.

**Comment by phooenixdassmatte** [comment]
> burrow tiles set children emergency
for setting the tiles in the children burrow to be the same as the tiles in the emergency burrow.

Cool :)
